zerver/lib/test_helpers.py

```diff
@@ -40,7 +40,7 @@
 
 if False:
     # Avoid an import cycle; we only need these for type annotations.
-    from zerver.lib.test_classes import ZulipTestCase, MigrationsTestCase
+    from zerver.lib.test_classes import ZulipTestCase, MigrationsTestCase  # type: ignore # ->warn-unreachable
```

Comment:
This should instead use the [`if TYPE_CHECKING` pattern](https://mypy.readthedocs.io/en/latest/common_issues.html#import-cycles) recommended by the documentation.

Reply:
Looks like that requires Python 3.5.2?  So should be OK for xenial.

Reply:
Cool, I'll apply this pattern then.